app/src/main/java/com/muzima/view/forms/FormTemplateWizardActivity.java

```diff
@@ -149,7 +150,7 @@ private boolean hasRegistrationFormSelected() {
         for (int i = 0; i < checkedItemPositions.size(); i++) {
             if (checkedItemPositions.valueAt(i)) {
                 AvailableForm selectedForm = (AvailableForm) listView.getItemAtPosition(checkedItemPositions.keyAt(i));
-                if (selectedForm.isRegistrationForm()) {
+                if (selectedForm.hasDescriminator(Constants.FORM_DISCRIMINATOR_REGISTRATION)) {
```

Comment:
Could you redefine the logic for determining whether it's a registration form within the isRegistrationForm() method? The reason is, that method can used in several places and therefore we need to remain consistent.